pkg/runtime/utils.go

```diff
@@ -28,7 +28,21 @@ import (
 	osi "github.com/sealerio/sealer/utils/os"
 )
 
-// LoadMetadata :read metadata via ClusterImage name.
+// Deprecated
```

Comment:
nice comment. Actually ,  we dont need Metafile anymore .

Reply:
So we can delete this code directly, all right?

Reply:
I have been tried to delete it, but it was used in buildah module, I think I can't change and verify it in a short time, cause it going to release.

Reply:
haha, next time we can optimize the code here

Reply:
I will keep this in my mind!